code/game/objects/effects/decals/turfdecal/markings.dm

```diff
@@ -225,170 +225,288 @@
 /obj/effect/turf_decal/mining/survival
 	icon_state = "survival"
 
-/// Automatically generates all subtypes for a siding with the given path.
-#define SIDING_SUBTYPE_HELPER(path)\
-##path/corner{\
-	icon_state = "siding_corner";\
-}\
-##path/end{\
-	icon_state = "siding_end";\
-}
+// /// Automatically generates all subtypes for a siding with the given path.
+// #define SIDING_SUBTYPE_HELPER(path)
+// ##path/corner{
+// 	icon_state = "siding_corner";
+// }
+// ##path/end{
+// 	icon_state = "siding_end";
+// } this is broken. also there was a backslash at the end of each line, was throwing errors when commented out. just fyi, future coder
```

Comment:
if it's broken, delete it. i thought it worked fine when i made it but just delete this if it's going to be completely unused